### PR TITLE
Add linters

### PR DESCRIPTION
**Pull request by mansa-susa**
## Hello Microverse Project
- In this Project I'm setting up Linters.
- Creating Branches
- Making Pull Request
- Set up Code Reviews

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,14 @@
...
+This project is to set up a "Hello World" repository. The goal is to show understanding of previous lessons learnt and tools, it does not require complex coding.
+- Branching
+- Setting up Linters
+- Fixing Lint errors
+- Requesting code reviews
+- Requesting pull requests
+- ## Built With
+- Html
+- Css
+- ## Live Demo
```
Comment (youmari) [inline README.md]: I see that you are removed some important headers, Please try to use this[ template](https://github.com/microverseinc/readme-template) and customize it to your own

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @mansa-susa,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great implementing `<h1>` header and CSS style 
✔️ No linters error 

## Required Changes ♻️

- [x] Your Title of your Pull Request is not descriptive enough, I suggest writing a meaningful title that explains what you've done in the PR
- [x] your short summary is missing, try to add a descriptive summary telling what changes you've done in more details,you can revisit these [rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL]  some commits are not capitalize, please next time try to follow the [rules ](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (mansa-susa) [reply README.md]: Yes i thought the other headers did not apply this particular project.

**Review by aimemalaika — APPROVED** [review]
Hi @mansa-susa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**